libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTracker.java

```diff
@@ -120,9 +119,18 @@
         THEMES_CUSTOMIZE_ACCESSED,
         THEMES_SUPPORT_ACCESSED,
         THEMES_DETAILS_ACCESSED,
+        SETTINGS_LANGUAGE_SELECTION_FORCED,
```

Comment:
That option is not part of the Site Settings, it's under Account Settings. I could change it to `ACCOUNT_SETTINGS_LANGUAGE_SELECTION_FORCED`.